WordPressAuthenticator/Unified Auth/View Related/2FA/TwoFAViewController.swift

```diff
@@ -41,18 +67,141 @@ final class TwoFAViewController: LoginViewController {
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        // TODO: update when real cells are added.
-        return 1
+        return rows.count
     }
 
-    /// Configure cells delegate method.
-    ///
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        // TODO: update when real cells are added.
-        return UITableViewCell()
+        let row = rows[indexPath.row]
+        let cell = tableView.dequeueReusableCell(withIdentifier: row.reuseIdentifier, for: indexPath)
+        configure(cell, for: row, at: indexPath)
+        return cell
+    }
+
+}
+
+// MARK: - Keyboard Notifications
+
+extension TwoFAViewController: NUXKeyboardResponder {
+    
+    @objc func handleKeyboardWillShow(_ notification: Foundation.Notification) {
+        keyboardWillShow(notification)
+    }
+
+    @objc func handleKeyboardWillHide(_ notification: Foundation.Notification) {
+        keyboardWillHide(notification)
+    }
+
+}
+
+private extension TwoFAViewController {
+
+    /// Registers all of the available TableViewCells.
+    ///
+    func registerTableViewCells() {
+        let cells = [
+            TextLabelTableViewCell.reuseIdentifier: TextLabelTableViewCell.loadNib(),
+            TextFieldTableViewCell.reuseIdentifier: TextFieldTableViewCell.loadNib(),
+            TextLinkButtonTableViewCell.reuseIdentifier: TextLinkButtonTableViewCell.loadNib()
+        ]
+        
+        for (reuseIdentifier, nib) in cells {
+            tableView.register(nib, forCellReuseIdentifier: reuseIdentifier)
+        }
+    }
+
+    /// Describes how the tableView rows should be rendered.
+    ///
+    func loadRows() {
+        rows = [.instructions, .code]
+
+        if errorMessage != nil {
+             rows.append(.errorMessage)
+         }
+
+        rows.append(.sendCode)
+    }
+
+    /// Configure cells.
+    ///
+    func configure(_ cell: UITableViewCell, for row: Row, at indexPath: IndexPath) {
+        switch cell {
+        case let cell as TextLabelTableViewCell where row == .instructions:
+            configureInstructionLabel(cell)
+        case let cell as TextFieldTableViewCell:
+            configureTextField(cell)
+        case let cell as TextLinkButtonTableViewCell:
+            configureTextLinkButton(cell)
+        case let cell as TextLabelTableViewCell where row == .errorMessage:
+            configureErrorLabel(cell)
+        default:
+            DDLogError("Error: Unidentified tableViewCell type found.")
+        }
+    }
+    
+    /// Configure the instruction cell.
+    ///
+    func configureInstructionLabel(_ cell: TextLabelTableViewCell) {
+        cell.configureLabel(text: WordPressAuthenticator.shared.displayStrings.twoFactorInstructions, style: .body)
+    }
+
+    /// Configure the textfield cell.
+    ///
+    func configureTextField(_ cell: TextFieldTableViewCell) {
+        cell.configureTextFieldStyle(with: .numericCode,
```

Comment:
Nit: I know this is pre-existing, but the external arguments `with` & `and` are not super-clear to understand. Feel free to ignore since it's not really part of this PR.

Reply:
Yea, I noticed that myself. I wasn't sure what the params were supposed to be. Since I've already branched from this branch, I'll take a look at improving that in a follow-up. Thanks @Gio2018 !